fido/update_signatures.py

```diff
@@ -159,5 +166,18 @@ def main(defaults=defaults):
         sys.exit('Aborting update...')
 
 
+def main():
+    """Main CLI entrypoint."""
+    parser = ArgumentParser(description='Download and convert the latest PRONOM signatures')
+    parser.add_argument('-tmpdir', default=options['tmp_dir'], help='Location to store temporary files', dest='tmp_dir')
+    parser.add_argument('-keep_tmp', default=options['deleteTempDirectory'], help='Do not delete temporary files after completion', dest='deleteTempDirectory', action='store_false')
+    parser.add_argument('-http_throttle', default=options['http_throttle'], help='Time (in seconds) to wait between downloads', type=float, dest='http_throttle')
+    args = parser.parse_args()
+    opts = defaults.copy()
+    opts.update(args.__dict__)
```

Comment:
I think [`vars(args)`](https://docs.python.org/3.5/library/functions.html#vars) is more appropriate here.  It accesses the namespace's dict, but without using a __ function.